### PR TITLE
refactor: github repo calls

### PR DESCRIPTION
**Pull request by jrasm91**

### CODE REVIEW

File: src/repositories/github.repository.ts [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
+import { Constants } from 'src/constants';
 import { IGithubInterface } from 'src/interfaces/github.interface';
 
 const octokit = new Octokit();
 
 export class GithubRepository implements IGithubInterface {
   private logger = new Logger(GithubRepository.name);
 
-  async getIssueOrPr(id: string) {
+  async getIssueOrPr(org: string, repo: string, id: string) {
```
Comment (danieldietzler) [inline src/repositories/github.repository.ts:12]: I don't *love* that the signature is different here, even though I know it doesn't matter

File: src/services/discord.service.ts [hunk 2]
```diff
@@ -100,7 +100,7 @@ export class DiscordService {
     const lastStarsCount = _star_history[channelId];
 
     try {
-      const starsCount = await this.github.getStarCount();
+      const starsCount = await this.github.getStarCount(GithubOrg.ImmichApp, GithubRepo.Immich);
```
Comment (danieldietzler) [inline src/services/discord.service.ts:103]: Having this be dynamic is cool. Could have an optional select thingy

**Review by danieldietzler — APPROVED** [review]

File: src/services/discord.service.ts (hunk 2 above)
Comment (jrasm91) [reply src/services/discord.service.ts:103]: Oh nice, yeah that would be cool. Or maybe have it default to something else in the immich-go channel, etc....

Comment (danieldietzler) [reply src/services/discord.service.ts:103]: Oooh channel specific, shiny!

**Pull request merged**